docs/bots.md

```diff
@@ -29,6 +29,53 @@ if (!hash_equals($digest, strtolower($_SERVER['HTTP_X_NEXTCLOUD_TALK_SIGNATURE']
 }
 ```
 
+## Bot added in a chat
+
+When the bot is added to a chat, the server sends a request to the bot, informing it of the event. The same signature/verification method is applied.
+
+### Headers
+
+| Header                            | Content type        | Description                                          |
+|-----------------------------------|---------------------|------------------------------------------------------|
+| `HTTP_X_NEXTCLOUD_TALK_SIGNATURE` | `[a-f0-9]{64}`      | SHA265 signature of the body                         |
+| `HTTP_X_NEXTCLOUD_TALK_RANDOM`    | `[A-Za-z0-9+\]{64}` | Random string used when signing the body             |
+| `HTTP_X_NEXTCLOUD_TALK_BACKEND`   | URI                 | Base URL of the Nextcloud server sending the message |
+
+### Content
+
+The content format follows the [Activity Streams 2.0 Vocabulary](https://www.w3.org/TR/activitystreams-vocabulary/).
+
+#### Sample Request
+
+```json
+{
+    "type": "Join",
+    "actor": {
+        "type": "Application",
+        "id": "bots/bot-a78f46c5c203141b247554e180e1aa3553d282c6",
+        "name": "Bot123"
+    },
+    "target": {
+        "type": "Collection",
+        "id": "n3xtc10ud",
+        "name": "world"
+    }
+}
+```
+
+#### Explanation
+
+| Path             | Description                                                                                                                                                                                                                                                                 |
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| actor.id         | Bot's [actor type](https://github.com/nextcloud/spreed/blob/main/docs/constants.md#actor-types-of-chat-messages) followed by the `/` slash character and a bot's unique sha1 identifier with `bot-` prefix.                                                                                               |
```

Comment:
Using a relative link will make it resolve automatically with in the generated docs:
```suggestion
| actor.id         | Bot's [actor type](constants.md#actor-types-of-chat-messages) followed by the `/` slash character and a bot's unique sha1 identifier with `bot-` prefix.                                                                                               |
```